parallel_algorithms/P3179/P3179.md

```diff
@@ -3855,16 +3866,16 @@ every iterator `i` in the range `[first1, last1)`. Otherwise, returns `false`.
 - [3.1]{.pnum} the types of `first1`, `last1`, `first2`, and `last2` meet the *Cpp17RandomAccessIterator* requirements
   ([random.access.iterators]{- .sref}) and `last1 - first1 != last2 - first2` for the overloads in namespace `std`;
 - [3.2]{.pnum} the types of `first1`, `last1`, `first2`, and `last2` pairwise model `sized_sentinel_for`
-  ([iterator.concept.sizedsentinel]{- .sref}) and `last1 - first1 != last2 - first2` for the first overload in namespace `ranges`,
+  ([iterator.concept.sizedsentinel]{- .sref}) and `last1 - first1 != last2 - first2` for the first overload in namespace `ranges`, [or]{.add}
 - [3.3]{.pnum} `R1` and `R2` each model `sized_range` and `ranges::distance(r1) != ranges::distance(r2)` for the second
   overload in namespace `ranges`,
```

Comment:
Should it now become "for the first and the third overloads in namespace ranges"? Similarly for [3.3], "for the second and the fourth overloads".

Reply:
Yes, I think it should. Done.